src/main/resources/api/api.yaml

```diff
@@ -150,6 +150,160 @@ paths:
+        - questions
+      parameters:
+        - name: body
+          in: body
+          required: true
+          description: Question that need to be added
+          schema:
+            $ref: '#/definitions/V1Question'
+      responses:
+        '200':
+          description: Return a new question
+          schema:
+            $ref: '#/definitions/V1Question'
+        '400':
+          description: Invalid input
+          schema:
+            $ref: '#/definitions/V1Error'
+        '403':
+          description: Access forbidden
+          schema:
+            $ref: '#/definitions/V1Error'
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/V1Error'
+    put:
+      summary: Update an existing question
+      operationId: updateQuestion
+      tags:
+        - questions
+      parameters:
+        - name: body
+          in: body
+          required: true
+          description: The id of the question to update
+          schema:
+            $ref: '#/definitions/V1Question'
+      responses:
+        '200':
+          description: Return the updated question
+          schema:
+            $ref: '#/definitions/V1Question'
+        '403':
+          description: Access forbidden
+          schema:
+            $ref: '#/definitions/V1Error'
+        '404':
+          description: Question not found
+          schema:
+            $ref: '#/definitions/V1Error'
+        '400':
+          description: Invalid input
+          schema:
+            $ref: '#/definitions/V1Error'
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/V1Error'
+  /v1/questions/{questionId}:
+    parameters:
+      - name: questionId
+        in: path
+        required: true
+        description: Id of the question to retrieve
+        type: integer
+        format: int64
+    get:
+      summary: Get the question with a specific id
+      operationId: showQuestionById
+      tags:
+        - questions
+      responses:
+        '200':
+          description: Return the question
+          schema:
+            $ref: '#/definitions/V1Question'
+        '400':
+          description: Invalid question ID supplied
+          schema:
+            $ref: '#/definitions/V1Error'
+        '404':
+          description: Question not found.
+          schema:
+            $ref: '#/definitions/V1Error'
+        '403':
+          description: Access forbidden
+          schema:
+            $ref: '#/definitions/V1Error'
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/V1Error'    
+    delete:
+      summary: Delete question with specific id
+      operationId: deleteQuestionById
+      tags:
+        - question
+      responses:
+        '400':
+          description: The specified question ID is invalid (not a number)
+          schema:
+            $ref: '#/definitions/V1Error'
+        '404':
```

Comment:
I think that a good response should be here 200 OK with empty body (like update)